idd/Energy+.idd.in

```diff
@@ -67789,6 +68421,8 @@ ThermalStorage:Ice:Simple,
       \type alpha
       \required-field
       \reference IceThermalStorageEquipment
+      \reference-class-name validBranchEquipmentTypes
```

Comment:
@mjwitte If there was a test file that shows the use of "ice storage objects" as a valid plant equipment list object, then the IP would have catch it, and fatal out. That was the case for some of the objects that I missed to include.